### ISSUE TITLE
 Make `rules_sass` available on registry.bazel.build

### ISSUE THREAD

**Issue by ztl8702** [bazelbuild/rules_sass#146]
(Just creating a TODO for now, not claiming to do it)

**Comment by ztl8702** [comment]
Does it need to be migrated to be based on `rules_js` first? (is it a good idea for https://github.com/bazelbuild
 rules to be based on aspect.dev maintained rules?)

It looks like it might simplify things

**Comment by Ubehebe** [comment]
Coincidentally, I ran into this yesterday too, when trying to migrate a repo from rules_nodejs to rules_js.

Both of the issues mentioned above (migrating to bzlmod and migrating to rules_js) are significant work. rules_sass hasn't seen real development for a couple of years, so I'm not sure if anyone is willing or able to do them.

On the other hand, the sass ruleset is simple. sass_library just aggregates sources and sass_binary invokes sass in a straightforward way. I forked the rule set and replicated its behavior, powered by rules_js, in an hour or two. My fork isn't public, but I'd open to sending it upstream if folks want.

**Comment by ztl8702** [comment]
yeah I realized it is easier to just run a genrule that invokes sass cli

**Comment by GZGavinZhao** [comment]
I created a Dart Sass-based `rules_sass` [here](https://github.com/GZGavinZhao/gzgz_rules_sass) that doesn't depend on NodeJS. Hopefully it would be useful to people.
